test/README.md

```diff
@@ -35,6 +35,9 @@ Our integration tests are based on functions from the samples project. To run in
 ### Extension Bundle
    If you've made any changes to the CSharp extension, you will need to run the CopySqlDllToExtensionBundle.Ps1 to update your local extension bundle so that the tests can run against your latest changes.
 
+### Running Java Tests
+   Run `mvn clean package` from the samples/samples-java and test/Integration/test-java folders to build the Java Azure Functions before running the tests.
```

Comment:
It'd be great to add a script that does this to make it a one step process (I'd even have it build both and then run the tests to make it more straightforward)